src/common/components/PageLayout.jsx

```diff
@@ -27,7 +27,8 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: 'column',
   },
   desktopDrawer: {
-    width: theme.dimensions.drawerWidthDesktop,
+    maxWidth: theme.dimensions.drawerWidthDesktop,
+    position: 'relative !important',
```

Comment:
btw why do we need this?

Reply:
otherwise the right content will use the full width and the wrapper will float on top of it.

Reply:
Emulator page also puts "position: relative".
For the !important, don't know why but it doesn't work without it.